starter/source/output/qaprint/st_qaprint_time_histories.F

```diff
@@ -68,6 +68,7 @@ SUBROUTINE ST_QAPRINT_TIME_HISTORIES(TH , ITHVAR , IPART  , SUBSETS,
         IF (OK_QA) THEN
 C
           !/TH
+          print*,'NTGGRP:',NITHGR,NTHGRP,SIZE(TH%ITHGRP)
```

Comment:
Hi, 
You left a debug print. Could you remove it?
Thank you, 
Laurent

Reply:
My Bad,
Removed.